src/main/java/com/scrive/pdftools/tools/LambdaInterface.java

```diff
@@ -0,0 +1,137 @@
+package com.scrive.pdftools.tools;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.scrive.pdftools.tools.flattener.FlattenerHandler;
+import com.scrive.pdftools.tools.utils.S3ForLambda;
+import org.apache.commons.io.IOUtils;
+import org.json.JSONObject;
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+
+public class LambdaInterface {
+    // Request parameters
+    private static final String RQ_PARAM_ACTION = "action";
+    private static final String RQ_PARAM_ACTION_FLATTEN = "flatten";
+    private static final String RQ_PARAM_ACTION_BODY = "body";
+    private static final String RQ_PARAM_S3_FILENAME = "s3FileName";
+
+    // Response parameters
+    private static final String RESP_FILE_NAME_PREFIX = "result_";
+    private static final String RESP_RESULT_S3_FILE_NAME = "resultS3FileName";
+    private static final String RESP_SUCCESS = "success";
+    private static final String RESP_ERROR = "error";
+    private static final String RESP_BODY = "body";
+    private static final String RESP_STATUS_CODE = "statusCode";
+    private static final String RESP_IS_B64_ENCODED = "isBase64Encoded";
+
+    private static void handleFlattenAction(JSONObject callJSON, OutputStream outputStream) {
+        getS3FileName(callJSON, outputStream).ifPresent(s3fileName -> {
+            getFileContent(s3fileName, outputStream).ifPresent(fileContent -> {
+                final JSONObject jsonObject;
+                try {
+                    jsonObject = new JSONObject(fileContent);
+                } catch (Exception e) {
+                    e.printStackTrace();
+                    LambdaInterface.respondWithError(outputStream, "Can't parse spec as JSON");
```

Comment:
let's keep the usage of `respondWith` and `respondWithError` within a single file consistent (`LambdaInterface.respond*` vs plain `respond*`). Since we are in the class itself, we can change all of those to the simpler (shorter) version.

Reply:
Of course. Nicely spotted.